tools/jslint.js

```diff
@@ -116,6 +96,41 @@ if (cluster.isMaster) {
     return;
   totalPaths = paths.length;
 
+  // Check if outputting JSLint report to a file instead of stdout
+  i = process.argv.indexOf('-o');
+  if (i !== -1) {
+    if (!process.argv[i + 1])
+      throw new Error('Missing output filename');
+    var outPath = process.argv[i + 1];
+    var count = 0;
+    if (!path.isAbsolute(outPath))
+      outPath = path.join(cwd, outPath);
+    if (format !== 'tap') {
+      fd = fs.openSync(outPath, 'w');
+    }
+    outFn = function(str) {
+      if (format === 'tap') {
+        const outPathNumbered = path.join(path.dirname(outPath),
+          `${path.basename(outPath, '.tap')}-${count}.tap`);
+        fd = fs.openSync(outPathNumbered, 'w');
```

Comment:
The file name is variable appending a count to it each time, I could definitely abstract the logic of building the base of the file name though.

As for the write, it is only writing once inside of the if statement... I should likely modify line 109 to be 'a'... that being said this code was carried over from the initial implementation.